### ISSUE TITLE
Internal error for one of my repos

### ISSUE THREAD

**Issue by jonathaningram** [schmittjoh/php-code-analysis#26]
http://jmsyst.com/reviews/0294a2a2-b3c0-4aad-a3b9-ded33e7e6e77

You may have already been notified of the app's stack trace, but if not, could you look into this?

**Comment by schmittjoh** [comment]
It looks like composer cannot install your package:

```
Loading composer repositories with package information
Installing dependencies
Nothing to install or update
Loading composer repositories with package information
Installing dev dependencies
Nothing to install or update
Generating autoload files



  [RuntimeException]
  Directory name must not be empty.
```

**Comment by jonathaningram** [comment]
Ok I'll see if it installs locally and fix the composer.json - can the error be returned as a user error on your end rather than an internal error?

**Comment by schmittjoh** [comment]
Yeah, that's a good idea, and it already does that for a couple of composer errors.

I'll see if we can add something for this error as well, not sure what the cause is here exactly.

**Comment by jonathaningram** [comment]
Yeah definitely think it's needed because my 2nd and 3rd reviews are now having internal server errors for some reason, but the first one didn't...

**Comment by schmittjoh** [comment]
There was a bug when a package had no dependencies. Hopefully, that fixed your other reviews as well.

I'll keep this open because better feedback in case of an error would definitely be useful.
